synapse/config/tls.py

```diff
@@ -118,7 +148,22 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         #
         tls_fingerprints: []
         # tls_fingerprints: [{"sha256": "<base64_encoded_sha256_fingerprint>"}]
-        """ % locals()
+
+        ## Support for ACME certificate auto-provisioning.
+        # acme:
```

Comment:
`acme_client_key` is missing from this list. Current convention is to list all the options here, even if they aren't really expected to be used much, since there's no other reference for the config options.